release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -48,9 +48,136 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
+#### Strand dump showing the total strand group and strand count in a Ballerina program
+
+The total number of strand groups and strands created in a Ballerina program is now shown in the strand dump.
+
+#### Allowing ambiguous target types in `fromJsonWithType` & `cloneWithType`
+
+When the user-specified target type was a union type consisting of multiple convertible types for a given source value, earlier `fromJsonWithType` & `cloneWithType` returned an error. Now for such user-specified target types, the operation succeeds without returning an error, returning a value whose inherent type is as follows. If the given source value belongs to at least one of the member types of the target union type, the inherent type of the returned value is the first found type to which the source value belongs. Otherwise, the inherent type of the returned value is the first convertible type in the union.
+
+For example, the following operations will not return an error.
+
+```ballerina
+type PetByAge record {
+    int age;
+    string nickname?;
+};
+
+type PetByType record {
+    "Cat"|"Dog" pet_type;
+    boolean hunts?;
+};
+
+type Pet PetByAge|PetByType;
+
+public function main() {
+    json j = {
+        "nickname": "Fido",
+        "pet_type": "Dog",
+        "age": 4
+    };
+    Pet|error pet = j.fromJsonWithType(); // inherent type of pet will be PetByAge
+
+    j = {x: 1};
+    record {float x;}|record {int x;}|error v = j.cloneWithType(); // inherent type of v will be record {int x;}
+}
+```
+
+#### Configurable support for the array of tables
+
+The configurable feature is improved to support the array of tables using a two-dimensional TOML array of in-line tables.
+
+For example, if the configurable variables are defined in the following way,
+
+```ballerina
+type Person record {
+    readonly int id;
+    string name;
+};
+
+configurable table<map<int>>[] array1 = ?;
+configurable table<Person> key (id)[] array2 = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+```toml
+array1 = [[{a = 1, b = 2}, {c = 3, d = 4}], [{e = 5, f = 6}]]
+
+array2 = [[{id = 1, name = "Anne"}, {id = 2, name = "Bob"}], [{id = 3, name = "Charles"}]]
+
+```
+
+#### Configurable support for nil type
+
+The configurable feature is now improved to support the following `()` cases
+
+- configurable variables of nil type
+  The configurable variables with nil type are now supported if the configuration is optional.
+
+For example, the following will be initialized with the default values without providing a compilation error.
+
+```ballerina
+configurable () nilValue = ();
+configurable ())[] nilArray = [];
+```
+
+- record fields with nil type
+  The record fields of nil type or a nilable union type are now supported by the configurable feature, only if the field contains a default value. If the configurable variable is of record type and the record contains nil or union of nil is now supported if the record configurable variable
```

Comment:
```suggestion
  The record fields of nil type or a nilable union type are now supported by the configurable feature only if the field contains a default value. If the configurable variable is of record type and the record contains nil or union of nil is now supported if the record configurable variable
```
Is it `nilable` or `nullable`?

The above sentence sounds incomplete. Need a comma for the if condition and the rest of the sentence should be completed.

Reply:
changed to `union of nil`